CT-1298 edit foi case closure reasons

misaka opened the pull request.

aliuk2012 commented on db/structure.sql:
```diff
@@ -2,8 +2,8 @@
--- Dumped from database version 9.5.5
--- Dumped by pg_dump version 9.5.5
+-- Dumped from database version 9.5.1
+-- Dumped by pg_dump version 9.5.1
```
why 9.5.1?

stephenrichards replied:
It's because it's the version I'm running at home.  The last time I tried to upgrade, it caused all sorts of  problems, so am loth to do it again until I have a week spare to sort it out!

This is only a problem because we use non standard plugins, and therefore have to dump the database structure as sql, rather than the rails standard.  We don't use this file anywhere.

misaka commented on lib/cts/demo_setup.rb:
```diff
@@ -24,24 +26,43 @@ def initialize(number)
             end
+          rescue => err
+            puts ">>>>>> ERROR #{err.class} #{err.message} <<<< "
+            puts "Unable to create that case"
```
Just some thoughts. It'd be nice to `ap kase` so that if anything got created, we could at least see what state it's in to understand why it couldn't be transitioned. I've been doing the same thing with the `create` sub-command, I also print out what cases have been created so we know how far it got.

misaka replied:
Ah, I see you've done this by adding progress reports below. Cool. :)

misaka committed:
CT-1298 allow editing of FOI closure details

misaka committed:
CT-1298 add event specs for old-style closed cases

misaka committed:
CT-1298 make exemptions reappear when editing closure info

misaka committed:
CT-1298 pull on close_page to edit_closure_page

On Travis for some reason we get an error that ClosePage doesn't exist when
trying to declare PageObjects::Pages::Cases::EditClosurePage

misaka committed:
CT-1298 update copy when case closure cant be edited

misaka committed:
CT-1298 fix check box generation for exemptions
